.claude/skills/release/SKILL.md

```
@@ -0,0 +1,69 @@
+Then perform these steps in order:
+
+1. Run `git checkout -b release/NEW_VERSION`
+
+2. Update the version in these files:
+   - `pubspec.yaml` — the `version` property
+   - `README.md` — the `ably_flutter: ^OLD_VERSION` line in the Installation section (update to `^NEW_VERSION`)
+
+3. Regenerate lock files:
+
+   a. Run `flutter pub get` in `example/` and `test_integration/` to update their `pubspec.lock` files:
+      ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)